resources/views/settings/language-preferences.blade.php

```diff
@@ -28,6 +28,7 @@
         <p>{{ __('The language you want to use for navigating this website.') }}</p>
 
         <div class="field @error('locale') field--error @enderror stack">
+            {{-- TODO: find a way to fix this required --}}
             <x-hearth-label for="locale" :value="__('Language') . ' ' . __('(required)')" />
```

Comment:
I think this works for all similar cases:

```suggestion
            <x-hearth-label for="locale"><x-required>{{ __('Language') }}</x-required></x-hearth-label>
```